src/static/css/openforms.css

```diff
@@ -558,6 +558,53 @@ button.datebutton:hover {
     box-shadow: 0 0 0 2px var(--ripple-primary) !important;
 }
 
+/* Form Formatting */
+.tablesaw-stack thead td,
+.tablesaw-stack thead th{
+    position: static !important;
+    font-size: unset !important;
+    overflow: unset !important;
+}
+
+.tablesaw-stack td,
+.tablesaw-stack th,
+.tablesaw-stack thead td,
+.tablesaw-stack thead th {
+    display: table-cell !important;
+    margin: 0 !important;
+}
+
+.tablesaw thead th{
```

Comment:
Can you please make the spaces after the selector consistent, e.g. '.tablesaw thead th {' rather than '.tablesaw thead th{'